data/sound-strings/af.json/localize.json

```diff
@@ -0,0 +1,108 @@
+{
+"make_a_slight_right_turn":"Keep right.",
+"make_a_slight_right_turn_street":"NULL",
+"make_a_slight_right_turn_street_verb":"NULL",
+"make_a_right_turn":"Turn right.",
+"make_a_right_turn_street":"NULL",
+"make_a_sharp_right_turn":"Sharp right.",
+"make_a_sharp_right_turn_street":"NULL",
+"enter_the_roundabout":"Enter the roundabout.",
+"enter_the_roundabout_street":"NULL",
+"enter_the_roundabout_street_verb":"NULL",
+"leave_the_roundabout":"Exit the roundabout.",
+"leave_the_roundabout_street":"NULL",
+"leave_the_roundabout_street_verb":"NULL",
+"make_a_slight_left_turn":"Keep left.",
+"make_a_slight_left_turn_street":"NULL",
+"make_a_slight_left_turn_street_verb":"NULL",
+"make_a_left_turn":"Turn left.",
+"make_a_left_turn_street":"NULL",
+"make_a_sharp_left_turn":"Sharp left.",
+"make_a_sharp_left_turn_street":"NULL",
+"make_a_u_turn":"Make a u-turn.",
+"make_a_u_turn_street":"NULL",
+"make_a_u_turn_street_verb":"NULL",
+"go_straight":"Go straight.",
+"exit":"Exit.",
+"onto":"onto",
+"take_exit_number":"Take exit",
+"take_exit_number_street_verb":"NULL",
+"route_recalculating":"Herbereken die roete",
```

Comment:
Language `af` has only one string translated `[route_recalculating]`. All other strings are from English. Is it correct? Should we keep `af` language?